torch/csrc/jit/passes/onnx/helper.h

```diff
@@ -14,6 +14,8 @@ static const int OPSET_VERSION_10 = 10;
 static const int OPSET_VERSION_11 = 11;
 static const int OPSET_VERSION_12 = 12;
 static const int OPSET_VERSION_13 = 13;
+static const int OPSET_VERSION_14 = 14;
```

Comment:
Not needed now, but maybe we should remove them at some point. Maybe use the constants defined in onnx.proto instead if we don't want to just use int literals.